scripts/ast-patches/allowAnyPropOnContainerChildren.js

```diff
@@ -0,0 +1,27 @@
+const ts = require("typescript");
+
+function allowAnyPropOnContainerChildren(ast) {
+  const $w = ast.statements.find(
```

Comment:
must of the code in the ast-patches are duplicated with minor changes. 
I would extract the login to one place

Reply:
i'll extract a few common queries...

Reply:
actually, it seems like this creates more mess imo, since every case is a bit different